doc/nrf/ug_thread_configuring.rst

```diff
@@ -137,6 +137,17 @@ You can choose to make it operate as Sleepy End Device (SED) by enabling the :op
 
 For more information, see `Device Types on OpenThread portal`_.
 
+.. _thread_ug_thread_1_2:
+
+Thread 1.2 options
+==================
+
+The OpenThread stack can be configured to operate in compliance with 1.1 or 1.2 Thread protocol version.
+By default |NCS| supports Thread 1.1 version, but you can change it using the following Kconfig options:
```

Comment:
Does this mean 1.1 version is enabled by default? If so, why do you provide an option for selecting 1.1?
Also, on the other pages you mention v1.1.1, here it's v1.1. This is inconsistent and confusing.

Reply:
Yes it is enabled by default. I provided this option, as CONFIG_OPENTHREAD_THREAD_VERSION_1_1 and CONFIG_OPENTHREAD_THREAD_VERSION_1_2 are available values of option CONFIG_OPENTHREAD_THREAD_VERSION, so I think it's consistent from the code point of view. I guess that in the future new values like CONFIG_OPENTHREAD_THREAD_VERSION_1_3 will be added and maybe another option will be set to default as v.1.1.1 will be obsolete. If it's not clear and will help somehow I can remove CONFIG_OPENTHREAD_THREAD_VERSION_1_1 from doc, but I think that it should exist in the Kconfig anyway. Using v.1.1.1 and v.1.1 is confusing for me too, as I found that both names are used in the project and I believe that in the context of same version. I think that v.1.1 should be used, as it's more common name in the project and also in official Thread Group materials.